src/content/operations/usage/_index.en.md

```diff
@@ -0,0 +1,854 @@
+kubectl apply -f ./nginx-service.yaml
+```
+
+When created, the Service is assigned a unique IP address (also called ClusterIP):
+
+```bash
+$ kubectl get service nginx
+NAME    TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)   AGE
+nginx   ClusterIP   100.2.29.136   <none>        80/TCP    13s
+```
+
+##### 6. Export the Service
+
+In order to signify that the Service should be visible and discoverable to other clusters in the clusterset, a `ServiceExport` needs to be
+created. The `subctl export` command can be used to automatically create the required `ServiceExport` object:
+
+```bash
+subctl export service --namespace default nginx
+Service exported successfully
+```
+
+##### 7. Test Service Discovery on **cluster3**
+
+First, verify that the exported `nginx` Service was imported to **cluster3** as expected. Submariner (via Lighthouse) automatically
+creates a corresponding `ServiceImport`:
+
+```bash
+$ kubectl get -n submariner-operator serviceimport
+NAME                     TYPE           IP               AGE
+nginx-default-cluster2   ClusterSetIP   [100.2.29.136]   62s
+```
+
+Next, run a test Pod on **cluster3** and try to access the `nginx` Service from within the Pod:
+
+```bash
+kubectl -n default  run --generator=run-pod/v1 \
+tmp-shell --rm -i --tty --image quay.io/submariner/nettest -- /bin/bash
+```
+
+```bash
+bash-5.0# curl nginx.default.svc.clusterset.local
+<!DOCTYPE html>
+<html>
+<head>
+<title>Welcome to nginx!</title>
+<style>
+    body {
+        width: 35em;
+        margin: 0 auto;
+        font-family: Tahoma, Verdana, Arial, sans-serif;
+    }
+</style>
+</head>
+<body>
+<h1>Welcome to nginx!</h1>
+<p>If you see this page, the nginx web server is successfully installed and
+working. Further configuration is required.</p>
+
+<p>For online documentation and support please refer to
+<a href="http://nginx.org/">nginx.org</a>.<br/>
+Commercial support is available at
+<a href="http://nginx.com/">nginx.com</a>.</p>
+
+<p><em>Thank you for using nginx.</em></p>
+</body>
+</html>
+```
+
+<!-- markdownlint-disable no-hard-tabs -->
+```bash
+bash-5.0# dig nginx.default.svc.clusterset.local
+; <<>> DiG 9.16.6 <<>> nginx.default.svc.clusterset.local
+;; global options: +cmd
+;; Got answer:
+;; WARNING: .local is reserved for Multicast DNS
+;; You are currently testing what happens when an mDNS query is leaked to DNS
+;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 55022
+;; flags: qr aa rd; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1
+;; WARNING: recursion requested but not available
+
+;; OPT PSEUDOSECTION:
+; EDNS: version: 0, flags:; udp: 4096
+; COOKIE: 6f9db9800a9a9779 (echoed)
+;; QUESTION SECTION:
+;nginx.default.svc.clusterset.local. IN	A
+
+;; ANSWER SECTION:
+nginx.default.svc.clusterset.local. 5 IN A	100.2.29.136
+
+;; Query time: 5 msec
+;; SERVER: 100.3.0.10#53(100.3.0.10)
+;; WHEN: Tue Dec 01 07:45:48 UTC 2020
+;; MSG SIZE  rcvd: 125
+```
+<!-- markdownlint-enable no-hard-tabs -->
+{{% notice note %}}
```

Comment:
This should be restrucutred to be something like a list to make it easier to read and understand. Maybe a separate section, like 
```
##### Service Discovery for Service in multiple clusters
If a service is exported from multiple clusters, IP returned follows these rules

* If service is exported from local cluster, it will always return IP of local service.
* If service is not exported locally, it will return service IP from one of the remote clusters where service is exported.
* If exported from multiple remote clusters, round robin is used to return IP from _connected_ clusters only. Any clusters currently in disconnected state are ignored by round robin.
* Users can access service from a specific cluster by prefixing service query with cluster-id `<cluster-id>.<svcname>.<namespace>.svc.clusterset.local`
```